read-the-docs/2/conf.py

```diff
@@ -93,6 +93,7 @@ def parse_project_name_from_pom():
     'portal-url':  ('https://developer.axonivy.com/portal/' + version + '/doc%s', None),
     'github-build-examples': ('https://github.com/axonivy/project-build-examples/blob/' + buildExampleVersion + '/compile-test%s', None),
     'api-browser-url': ('https://developer.axonivy.com/api-browser?configUrl=https://developer.axonivy.com/doc/' + version + '/openapi/config.json&urls.primaryName=%s', None),
+    'project-build-plugin-doc': ('https://axonivy.github.io/project-build-plugin/release/' + version + '/%s', None),
```

Comment:
I'm not sure whether this cahnge make sense:
'release' is also dynamic in here ... 'snapshot/version' is the other valid alternative. 
so we will have a linking problem if we define the default version which is a 'snapshot'.
...furthermore the version seems to relate to the 'ivy-version. But there is yet no guarantee that the project-build-plugin will stay sinc with the ivy-core. It's in theory a completely independent product lifecycle

Reply:
- in my opinion it's really bad if someone links to a snapshot release! so release makes totally sense to me
- you're right that this can be different, I only liked the idea that would always link to the latest project build plugin... but maybe your right that it's better if the version is not in sync...

Reply:
however if we can't introduce a version that actually fits: I'd prefer to have a static URI, without any versions > and the github.io site will automatically redirect to the latest release > https://axonivy.github.io/project-build-plugin

Reply:
I've created an additional subtask for this version evaluation